src/commands/appdistribution-distribute.ts

```diff
@@ -73,14 +54,14 @@ module.exports = new Command("appdistribution:distribute <release-binary-file>")
     const appId = getAppId(options.app);
     const distribution = new Distribution(file);
     const releaseNotes = getReleaseNotes(options.releaseNotes, options.releaseNotesFile);
-    const testers = getTestersOrGroups(options.testers, options.testersFile);
-    const groups = getTestersOrGroups(options.groups, options.groupsFile);
-    const requests = new AppDistributionClient(appId);
+    const testers = testerEmailParser(options.testers, options.testersFile);
```

Comment:
The name of the function `testerEmailParser` should be more generalize since it can be used for more than just parsing tester emails. E.g. on the line below, it's used to get a list of groups.

Reply:
+1. I recommend combining testerEmailParser and ensureFileExists into one helper class, something like `options-parser-util.ts`